source/brailleTables.py

```diff
@@ -69,6 +77,33 @@ class BrailleTable(NamedTuple):
 and cleared when calling L{terminate}.
 """
 
+_inputTableForLangs = dict()
+"""Maps languages to input L{BrailleTable.fileName}."""
+
+_outputTableForLangs = dict()
+"""Maps languages to output L{BrailleTable.fileName}."""
+
+
+def getDefaultTableForCurLang(tableType: TableType) -> str:
+	"""Gets the file name of the braille table for the current NVDA language.
+	:param tableType: input or OUTPUT.
+	:return: A L{BrailleTable} fileName.
+	"""
+	match tableType:
+		case tableType.INPUT:
+			langDict = _inputTableForLangs
+		case tableType.OUTPUT:
+			langDict = _outputTableForLangs
+		case _:
+			raise ValueError(f"Unknown tableType: {tableType}")
+	lang = languageHandler.getLanguage()
+	table = langDict.get(lang)
+	if table is not None:
+		return table
+	if "_" in lang:
+		lang = lang.split("_")[0]
+	return langDict.get(lang, "en-ueb-g1.ctb")
```

Comment:
can you please turn this into a constant?

```suggestion
	return langDict.get(lang, DEFAULT_TABLE)
```